src/lib/cache.svelte.ts

```
@@ -202,11 +210,57 @@ async function initialize() {
+	const selection = dishes_.filter(dish => !dishes[dish]) // get only the dishes we don't already have
+	
+	// Dishes
+	const {data: dishesData, error: dishesError} = await supabase
+		.from("dishes")
+		.select("id, name, manager")
+		.in("id", selection)
+	if (dishesError) { console.error("Error in getting dishes:", dishesError); toast.error("Error in getting dishes."); return }
+	dishesData.forEach(dish => dishes[dish.id] = {
+		...dish,
+		ingredients: {},
+	})
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)